README.md

```diff
@@ -38,12 +54,15 @@ We recommend using the Managed Config Controller service which comes bundled wit
 
 See the Google Cloud [quickstart](https://cloud.google.com/anthos-config-management/docs/tutorials/manage-resources-config-controller) guide for getting up and running with Config Controller
 
-A setup script is provided in the repository [gcp-tools](https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/setup-kcc.sh) that will configure the Config Controller cluster.  The instructions in the [Advanced Install](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/advanced-install.md) are automated as part of the [setup-kcc.sh](https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/setup-kcc.sh) script.
```

Comment:
looks good, but we need a reference somewhere on the main page to the setup-kcc.sh script.
A large portion of the single tenant installs of the landing zone opt for automated config-controller deployment bootstrap and not the instructions on the advanced-install or the more current docs/landing-zone-v2 readme

Both of these need to be on the main page - but we can remove the double link to setup-kcc.sh

automated
https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/README.md
and manual
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md#setup

Reply:
actually the landing-zone-v2 readme has a reference to the config controller automation script
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md#initial-organization-configuration

but ideally the main readme here - does as well

Reply:
Stan, you are free to merge your own PRs in your branch and adjust there in the gcp brqnch - before a formal iterated set of PRs from the gcp branch to main and a full team review.